apps/omg_child_chain/lib/omg_child_chain/sync_supervisor.ex

```diff
@@ -38,7 +38,19 @@ defmodule OMG.ChildChain.SyncSupervisor do
   end
 
   def init(args) do
-    opts = [strategy: :one_for_one]
+    # Assuming the values max_restarts and max_seconds,
+    # then, if more than max_restarts restarts occur within max_seconds seconds,
+    # the supervisor terminates all child processes and then itself.
+    # The termination reason for the supervisor itself in that case will be shutdown.
+    # max_restarts defaults to 3 and max_seconds defaults to 5.
+
+    # We have 9 children, roughly 7 of them have a dependency to the internetz.
+    # The internetz is flaky. We account for that and allow the flakyness to pass
+    # by increasing the restart strategy. But not too much, because if the internetz is
+    # really off, HeightMonitor should catch that in max 8 seconds and raise an alarm.
+    max_restarts = 3
+    max_seconds = 5
+    opts = [strategy: :one_for_one, max_restarts: max_restarts * 5, max_seconds: max_seconds]
```

Comment:
> HeightMonitor should catch that in max 8 seconds and raise an alarm.

But you're still setting this to max 5 seconds. Is it okay to not be > 8?

Reply:
the thing is -the timer resets itself if you don't reach `max_restarts ` in `max_seconds`. so it's a bit difficult to figure out what the correct numbers are - for example, a delay in a process `init/1` could reset the timer I'm mentioning.

Reply:
> the timer resets itself if you don't reach max_restarts in max_seconds. 

oh wow.....I see. I thought if force failure as long as it runs overtime.